### PR TITLE
LVUIDF-66:Added initial TinyMCE

### PR DESCRIPTION
**Pull request by Fleeck**
Safe copy of https://github.com/dark-side/language-school/pull/41, reworked.

https://ssu-jira.softserveinc.com/browse/LVUIDF-66

 - [x] Removed 'path' parameter from Page model.
 - [x] Implemented TinyMCE editor into Page.

To review:
- [x] @rtriska
- [ ] @forecome 

To merge:
- [ ] @izagor
- [ ] @forecome

### CODE REVIEW

File: db/migrate/20150710080653_create_pages.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def change
       t.string :title
       t.text :body
       t.string :path
-	  
+
```
Comment (Fleeck) [inline db/migrate/20150710080653_create_pages.rb]: it was an extra trailing space.

File: app/assets/javascripts/tinymce_custom_options.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+window.tinymce_custom_options = {
+    selector: "textarea#body",
```
Comment (rtriska) [inline app/assets/javascripts/tinymce_custom_options.js]: I asked you to move this parameter. This file should contain only general parameters for all editors in application.

**Comment by rtriska** [comment]
Suggestion regarding tests. Use only one expectation per one tests. It makes sence. What if one expectation is failing and another one no. We should prevent such situations.

**Comment by Fleeck** [comment]
@rtriska 
Reworked according to your suggestions. Please, review once more.

**Comment by rtriska** [comment]
Good :green_apple:

**Pull request merged**